fpga_interchange/populate_chip_info.py

```diff
@@ -1640,6 +1641,65 @@ def build_input_site_port(self, tile_type_idx, port_name, site_wire_name,
         return site_wire_idx
 
 
+def populate_macros(device, chip_info):
+    prims = device.get_primitive_library()
+
+    def get_cell(cell_type):
+        # Gets the definition for a cell
+        for lib in prims.libraries.values():
+            if cell_type in lib.cells:
+                return lib.cells[cell_type]
+        return None
+
+    if 'macros' in prims.libraries:
```

Comment:
For readability might be better to return in case macros is not in `prims.libraries`, so we can remove an indentation level

Reply:
Fixed